### PR TITLE
Edge Gateway DHCPv6 and a few IPv6 settings

### PR DESCRIPTION
**Pull request by Didainius**
This PR adds IPv6 related component testing and a few missing features:
* NSX-T Edge Gateway SLAAC Profile (the name for Edge Gateway DHCPv6 configuration in UI)
* NSX-T ALB Virtual Service IPv6 Virtual IP support and test
* Dual Stack mode support for Org VDC network type `types.OpenApiOrgVdcNetwork` via field `EnableDualSubnetNetwork`

### CODE REVIEW

**Review by lvirbalas — APPROVED** [review]

File: .changes/v2.21.0/587-features.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* Added NSX-T Edge Gateway SLAAC Profile (DHCPv6) configuration support `NsxtEdgeGateway.GetSlaacProfile` and
+  `NsxtEdgeGateway.UpdateSlaacProfile` [GH-587]
```
Comment (adambarreiro) [inline .changes/v2.21.0/587-features.md:2]:
```suggestion
  `NsxtEdgeGateway.UpdateSlaacProfile` [GH-582]
```

**Review by adambarreiro — APPROVED** [review]
LGTM

File: .changes/v2.21.0/587-improvements.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+* NSX-T ALB Virtual Service supports IPv6 Virtual Service using field`IPv6VirtualIpAddress` in
+  `types.NsxtAlbVirtualService` for VCD 10.4.0+ [GH-587]
+* Add field `EnableDualSubnetNetwork` to enable Dual-Stack mode for Org VDC networks in
+  `types.OpenApiOrgVdcNetwork` [GH-587]
```
[the remark this answers is not in the document]
Comment (Didainius) [reply .changes/v2.21.0/587-improvements.md]: Fixed.

File: .changes/v2.21.0/587-features.md (hunk 1 above)
Comment (Didainius) [reply .changes/v2.21.0/587-features.md]: Fixed.

**Review by adezxc — APPROVED** [review]

**Review by dataclouder — APPROVED** [review]

**Pull request merged**